portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
```

Comment:
This title should probably be different for this page, right?

Reply:
Changed title